src/erlcloud_cloudfront.erl

```diff
@@ -0,0 +1,243 @@
+-module(erlcloud_cloudfront).
+
+-include_lib("xmerl/include/xmerl.hrl").
+
+%% Library initialization.
+-export([configure/2, configure/3, new/2, new/3]).
+
+-export([list_distributions_all/0, list_distributions_all/1,
```

Comment:
well, all EC2 module filled with _all functions. Do you propose just to remove this "get all results" stuff and pagination for user?